### ISSUE TITLE
Reminder users to shut down TemplateVM in order for changes to take effect

### ISSUE THREAD

**Issue by andrewdavidwong** [QubesOS/qubes-issues#6056]
**The problem you're addressing (if any)**

This sort of problem is common:

1. User has a `work` TemplateBasedVM based on the `fedora-32` TemplateVM.
2. User starts `fedora-32` and installs a new application in it.
3. User (re)starts `work`.
4. User wonders why the new application is nowhere to be found in `work`.

Answer: User forgot to shut down `fedora-32` between steps 2 and 3.

We still get many questions along the lines of "Where is my application?" and this is usually the answer.

**Describe the solution you'd like**

Our current solution is the one described in #5657. However, it does not seem to be enough, because I still see a lot of questions about it, especially from new users.

I don't know what the best solution would be, but here's one idea:

1. User has a `work` TemplateBasedVM based on the `fedora-32` TemplateVM.
2. User starts `fedora-32` and installs a new application in it.
3. User attempts to (re)start `work`.
4. User gets a prompt that says something like:
> It looks like you're trying to start `work`, which is based on `fedora-32`. Since `fedora-32` is still running, any changes you've made to it won't be reflected in `work` until `fedora-32` is shut down. Would you like to shut down `fedora-32` before starting `work`?
> 
> If you select **yes**, `fedora-32` will be shut down first, then `work` will start.  
> If you select **no**, `fedora-32` will be left running, and `work` will start immediately.

If a pop-up is too disruptive, then perhaps a passive notification with similar language instead.

**Where is the value to a user, and who might that user be?**

Less experienced users will be less likely to forget to shut down their TemplateVMs, or more generally, less likely to be confused about why changes in templates aren't reflected in TemplateBasedVMs.

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

https://www.qubes-os.org/doc/templates/

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

#5657

CC: @marmarta, @ninavizz

**Comment by sptankard** [comment]
I don't know if you've been encountering questions in the wild about it, but i could forsee similar issues happening with DVM templates. (But maybe only more technical users make any customizations to dvm templates.)

I think the semantic distinction of DVM templates is actually more subtle and should probably be reflected in the UI more generally, as a failed understanding can have security implications.

**Comment by marmarta** [comment]
this is in practice a duplicate of https://github.com/QubesOS/qubes-issues/issues/8070
